client/gc_client.go

```diff
@@ -0,0 +1,105 @@
+package pd
+
+import (
+	"context"
+	"time"
+
+	"github.com/opentracing/opentracing-go"
+	"github.com/pingcap/kvproto/pkg/pdpb"
+	"github.com/pingcap/log"
+	"github.com/tikv/pd/client/grpcutil"
+	"go.uber.org/zap"
+)
+
+// GCClient is a client for doing GC
+type GCClient interface {
```

Comment:
Don't we need a `GetGCSafePointV2` method?

Reply:
In the implementation of safepoint v1, GetGCSafePin is also not implemented in client.go.
Each component uses `UpdateGCSafePoint(safepoint=0)` to obtain the value of GC SafePoint in v1.

Is it necessary for us to add a GetGCSafePointV2 function?
@MyonKeminta

Reply:
In safepoint v1,BR use UpdateGCSafePoint(safepoint=0) to get gc safepoint and check `backups>gc safepoint is secure ` now,this operation is redundant, it's need a tidb pr to resolve it.

in safepoint v1, should directly determine if UpdateServiceGCSafePoint returns success, which indicates that the data specified in backups is secure.